### PR TITLE
Ignore openshift-logging and improve setup

### PR DESCRIPTION
**Pull request by porridge**
https://redhat.pagerduty.com/incidents/Q0WPB1F0AO8459

### CODE REVIEW

**Review by dhaus67 — APPROVED** [review]

**Review by stehessel — CHANGES_REQUESTED** [review]
This has to be changed in https://github.com/stackrox/rhacs-observability-resources/blob/master/resources/mixins/kubernetes/mixin.libsonnet, the yaml is auto generated.

**Comment by porridge** [comment]
@stehessel can you please help me with this? When running `make generate` on clean `master` I'm getting the following error. FWIW, my `jsonnet` is `Jsonnet commandline interpreter v0.20.0` installed by adding `pkgs.jsonnet` to `flake.nix` :thinking:


```
[mowsiany@mowsiany rhacs-observability-resources]$ make generate 
make -C resources/mixins/kubernetes generate
make[1]: Entering directory '/home/mowsiany/go/src/github.com/stackrox/rhacs-observability-resources/resources/mixins/kubernetes'
RUNTIME ERROR: couldn't open import "kubernetes-mixin/mixin.libsonnet": no match locally or in the Jsonnet library paths.
	mixin.libsonnet:2:20-61	thunk <kubernetes>
	mixin.libsonnet:4:1-11	thunk <import>
	<cmdline>:1:22-46	thunk <value>
	std.jsonnet:1278:9-14	thunk <v>
	std.jsonnet:1199:10	thunk <a>
	std.jsonnet:1199:10-19	function <anonymous>
	std.jsonnet:1199:10-19	function <aux>
	std.jsonnet:1278:5-23	function <anonymous>
	<cmdline>:1:1-65	
make[1]: *** [Makefile:10: alerts] Error 1
make[1]: Leaving directory '/home/mowsiany/go/src/github.com/stackrox/rhacs-observability-resources/resources/mixins/kubernetes'
make: *** [Makefile:3: generate] Error 2
[mowsiany@mowsiany rhacs-observability-resources]$ 
```

**Comment by stehessel** [comment]
I think you need

```
pkgs.go-jsonnet
pkgs.jsonnet-bundler
```

You might have to run `jb install github.com/kubernetes-monitoring/kubernetes-mixin` as well, not sure if this is required or will happen automatically if uninstalled.

**Comment by porridge** [comment]
Thanks, done, I also I added instructions. PTAL.

File: README.md [hunk 1]
```diff
@@ -41,6 +41,10 @@ To make changes to the rhacs dashboards:
 * Run `make generate` to generate the corresponding resources for the Grafana operator.
 
 To make changes to Kubernetes mixin resources:
+
+First, make sure you have `go-jsonnet` and `jsonnet-bundler` installed, and run `(cd resources/mixins/kubernetes && jb install github.com/kubernetes-monitoring/kubernetes-mixin)` to set things up.
```
Comment (stehessel) [inline README.md:45]: I think is idempotent. So we can just add it to the `make generate`/`make update` commands instead of documenting them.

Comment (porridge) [reply README.md]: Great idea, it seems to be very fast on re-run. PTAL

**Review by stehessel — APPROVED** [review]

**Pull request merged**